.golangci.yml

```diff
@@ -1,7 +1,7 @@
 ---
 run:
   # concurrency: 6
-  timeout: 1m
+  timeout: 2m
```

Comment:
Question... are we starting to time out on golangci-lint ? (just curious).

Reply:
Yes, sometimes on CI `golangci` achieves 1m time limit

Reply:
Good to know, obviously your solution here is the right one :)